content/millennium/r4/financial/general/charge-item.md

```diff
@@ -157,6 +157,53 @@ List an individual ChargeItem by its id:
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-credit
+
+<%= beta_tag(action: true) %>
+
+Creates an offsetting ChargeItem for an existing debit ChargeItem.
+
+    POST/ChargeItem/:id/charge-item-credit
```

Comment:
Should we be doing something like this
`POST  /ChargeItem/:id/$credit` as followed by Binary CCD custom operation
![image](https://user-images.githubusercontent.com/33313942/132175102-34c1c6a5-7123-494b-a766-815e05a9352b.png)
can anyone confirm ? @SantanuBanerjee / @alexbeckner1

Reply:
yes, I think that is correct. We can change it to $credit

Reply:
Done: https://github.com/cerner/fhir.cerner.com/pull/726/commits/71fbf617bf8defa672134577e046925aae99cedd#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR166